projects/ziti-console-lib/src/lib/features/projectable-forms/service/service-form.service.ts

```diff
@@ -497,6 +504,30 @@ export class ServiceFormService {
         return data;
     }
 
+    validateConfigItems(items, parentType = 'object') {
+        let isValid = true;
+        items.forEach((item) => {
+            if (item.type === 'array') {
+                if (item?.component?.instance?.isValid) {
```

Comment:
`item?.component?.instance?.isValid` is duplicated in this one else and also a bunch in projects/ziti-console-lib/src/lib/services/schema.service.ts...

can't this be made into a reusable function? feels like too much copy/paste code for my taste... :/

Reply:
Updated to prevent copied/repeated code